### ISSUE TITLE
Compiler crashes

### ISSUE THREAD

**Issue by juriansluiman** [e-jigsaw/gulp-riot#4]
``` cmd
vagrant@demo:/vagrant$ gulp
[21:14:13] Using gulpfile /vagrant/gulpfile.js
[21:14:13] Starting 'default'...
[21:14:13] Finished 'default' after 21 ms

/vagrant/node_modules/gulp-riot/build/index.js:19
      file.contents = new Buffer(compiler(file.contents.toString()));
                                 ^
TypeError: object is not a function
    at DestroyableTransform.transform [as _transform] (/vagrant/node_modules/gulp-riot/build/index.js:19:34)
    at DestroyableTransform.Transform._read (/vagrant/node_modules/gulp-riot/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/vagrant/node_modules/gulp-riot/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/vagrant/node_modules/gulp-riot/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/vagrant/node_modules/gulp-riot/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:227:5)
    at DestroyableTransform.Writable.write (/vagrant/node_modules/gulp-riot/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:194:11)
    at write (/vagrant/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:623:24)
    at flow (/vagrant/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:632:7)
    at DestroyableTransform.pipeOnReadable (/vagrant/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:664:5)
    at DestroyableTransform.emit (events.js:92:17)
```

Using gulpfile:

``` js
var gulp   = require('gulp'),
    riot   = require('gulp-riot'),
    chmod  = require('gulp-chmod');

gulp.task('default', function() {
    gulp.src('public/scripts/src/comments.tag')
        .pipe(riot())
        .pipe(chmod(644))
        .pipe(gulp.dest('public/scripts/dist'));

    return;
});
```

Am I doing something wrong? `riot public/scripts/src/comments.tag public/scripts/dist` does compile the file.

**Comment by isnifer** [comment]
+1

**Comment by isnifer** [comment]
@juriansluiman try my solution in:

``` sh
./node_modules/gulp-riot/build/index.js
```

**Comment by e-jigsaw** [comment]
ok, published `0.1.3`.

try new version!
